src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -213,7 +213,7 @@ package com.videojs.providers{
         }
         
         public function get playing():Boolean{
-            return _isPlaying;
+            return _playbackStarted;
```

Comment:
@bclwhitaker yeah `_isPlaying && _isPaused` is ridiculous. In core we have the `vjs-has-started` class if you're looking for similar naming, though I've considered changing that to `vjs-played` to match `vjs-ended` and the video element's `played` property. What you have here is good though.